js/index.js

```diff
@@ -37,6 +37,152 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
+// navBar build
+let servicesNav = document.querySelectorAll('.container nav a')[0];
```

Comment:
really clean and easy way to do this, I would suggest that when you see multiple elements that are making your type the same things over and over again, start looking for a function to handle the heavy lifting for you.